src/providers/zoom.js

```diff
@@ -0,0 +1,22 @@
+export default function Zoom(options) {
+  return {
+    id: "zoom",
+    name: "Zoom",
+    type: "oauth",
+    version: "2.0",
+    params: { grant_type: "authorization_code" },
+    accessTokenUrl: "https://zoom.us/oauth/token",
+    authorizationUrl:
+      "https://zoom.us/oauth/authorize?response_type=code",
+    profileUrl: "https://api.zoom.us/v2/users/me",
+    profile(profile) {
+      return {
+        id: profile.id,
+        name: `${profile.first_name} ${profile.last_name}`,
+        email: profile.email,
+      }
+    },
+    protection: ["pkce", "state"]
```

Comment:
We're missing a `,` here, hence the [build is failing](https://github.com/nextauthjs/next-auth/pull/2110/checks?check_run_id=2725612640) 💅🏽